src/lib.rs

```diff
@@ -850,6 +850,14 @@ impl<T: Counter> Histogram<T> {
         self.record_n(value, T::one())
     }
 
+    /// Unrecord `value` in the histogram, removing from the value's current count.
```

Comment:
I'd prefer to do this change in a PR that's separate from all the clippy/lint fixes. Could you split them up please?